src/K9OCRS/ClientApp/src/shared/components/SideNav.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const SideNav = (props) => {
+  const current = props.current;
+  const userRole = props.userRoleID;
+  var active = [false, false, false, false, false];
+  active[current] = true;
+  var hasSections = false;
+  if (userRole < 3) {
+    hasSections = true;
+  }
+
+  return (
+    <div
+      className='nav flex-column nav-pills'
+      role='tablist'
+      aria-orientation='vertical'
+    >
+      <a
+        className={`nav-link ${active[0] ? 'active' : ''}`}
+        data-toggle='pill'
+        href='/Account'
+        role='tab'
+      >
+        Profile
+      </a>
+      <a
+        className={`nav-link ${active[1] ? 'active' : ''}`}
+        data-toggle='pill'
+        href='/MyDogs'
```

Comment:
Broken link, gotta add "/Account" to the path.
```suggestion
        href='/Account/MyDogs'
```